### ISSUE TITLE
New Resource Status Page shows only one host per service when using host groups

### ISSUE THREAD

**Issue by srinathman** [centreon/centreon#9474]
```
# rpm -qa | grep centreon | egrep -v "(plugin|pack)" | sort
centreon-20.10.3-10.el7.centos.noarch
centreon-auto-discovery-server-20.10.2-1.el7.centos.noarch
centreon-base-config-centreon-engine-20.10.3-10.el7.centos.noarch
centreon-broker-20.10.3-5.el7.centos.x86_64
centreon-broker-cbd-20.10.3-5.el7.centos.x86_64
centreon-broker-cbmod-20.10.3-5.el7.centos.x86_64
centreon-broker-core-20.10.3-5.el7.centos.x86_64
centreon-broker-storage-20.10.3-5.el7.centos.x86_64
centreon-clib-20.10.0-2.el7.centos.x86_64
centreon-common-20.10.3-10.el7.centos.noarch
centreon-connector-20.10.0-2.el7.centos.x86_64
centreon-connector-perl-20.10.0-2.el7.centos.x86_64
centreon-connector-ssh-20.10.0-2.el7.centos.x86_64
centreon-database-20.10.3-10.el7.centos.noarch
centreon-engine-20.10.2-2.el7.centos.x86_64
centreon-engine-daemon-20.10.2-2.el7.centos.x86_64
centreon-engine-extcommands-20.10.2-2.el7.centos.x86_64
centreon-fingerprint-20.04.0-1.el7.centos.x86_64
centreon-gorgone-20.10.2-1.el7.centos.noarch
centreon-gorgone-centreon-config-20.10.2-1.el7.centos.noarch
centreon-license-manager-20.10.1-1.el7.centos.noarch
centreon-license-manager-common-20.10.1-1.el7.centos.noarch
centreon-nrpe-daemon-2.15-4.el7.centos.x86_64
centreon-open-tickets-20.10.1-2.el7.centos.noarch
centreon-perl-libs-20.10.3-10.el7.centos.noarch
centreon-poller-centreon-engine-20.10.3-10.el7.centos.noarch
centreon-pp-manager-20.10.0-1.el7.centos.noarch
centreon-release-20.10-2.el7.centos.noarch
centreon-trap-20.10.3-10.el7.centos.noarch
centreon-web-20.10.3-10.el7.centos.noarch
centreon-widget-engine-status-20.10.0-5.el7.centos.noarch
centreon-widget-global-health-20.10.0-5.el7.centos.noarch
centreon-widget-graph-monitoring-20.10.0-5.el7.centos.noarch
centreon-widget-grid-map-20.10.0-5.el7.centos.noarch
centreon-widget-hostgroup-monitoring-20.10.0-5.el7.centos.noarch
centreon-widget-host-monitoring-20.10.0-5.el7.centos.noarch
centreon-widget-httploader-20.10.0-5.el7.centos.noarch
centreon-widget-live-top10-cpu-usage-20.10.0-5.el7.centos.noarch
centreon-widget-live-top10-memory-usage-20.10.0-5.el7.centos.noarch
centreon-widget-servicegroup-monitoring-20.10.0-5.el7.centos.noarch
centreon-widget-service-monitoring-20.10.0-5.el7.centos.noarch
centreon-widget-tactical-overview-20.10.0-5.el7.centos.noarch

```
***Operating System***
CentOS,

***Browser used***
-  Google Chrome

### Description
New Resource Status Page shows only one host per service when using host groups

### Steps to Reproduce
1. Create a Host Group
2. Create a Service Associated to the Host Group
3. Go to "Monitoring" -> "Resource Status"
4. Select new filter
5. Click Arrow down to expand the criterias  section
5. In Resource select "Service"  and leave all others 'empty'

### Describe the received result
Instead of seeing ALL hosts and ALL services (Like in the old services page), each different service is listed only once for a 'random' host.

### Describe the expected result
All services, for all hosts should be listed

![image](https://user-images.githubusercontent.com/2333250/107593873-78e33500-6bde-11eb-8f57-27951b75925d.png)

**Comment by xfouloux** [comment]
I can config it's happening, using the deprecated view list all services by hosts.

**Comment by cgagnaire** [comment]
Duplicate of https://github.com/centreon/centreon/issues/9180
